.travis.yml

```diff
@@ -12,21 +12,30 @@ opam: &OPAM
       export PS4='+ \e[33;1m(\$0 @ line \$LINENO) \$\e[0m '
       set -ex  # -e = exit on failure; -x = trace for debug
       opam update -y
+      opam remove -y coq-bignums  # remove the coq-bignums from the docker image
       opam pin add ${CONTRIB_NAME} . -y -n -k path
       opam install ${CONTRIB_NAME} -y -j ${NJOBS} --deps-only
       opam config list
       opam repo list
       opam list
       "
   script:
-  - echo -e "${ANSI_YELLOW}Building ${CONTRIB_NAME}...${ANSI_RESET}" && echo -en 'travis_fold:start:script\\r'
+  - echo -e "${ANSI_YELLOW}Building and testing ${CONTRIB_NAME}...${ANSI_RESET}" && echo -en 'travis_fold:start:script\\r'
   - |
     docker exec COQ /bin/bash --login -c "
       export PS4='+ \e[33;1m(\$0 @ line \$LINENO) \$\e[0m '
       set -ex
       sudo chown -R coq:coq /home/coq/${CONTRIB_NAME}
       opam install ${CONTRIB_NAME} -v -y -j ${NJOBS}
       "
+  - |
+    docker exec COQ /bin/bash --login -c "
+      export PS4='+ \e[33;1m(\$0 @ line \$LINENO) \$\e[0m '
+      set -ex
+      sudo chown -R coq:coq /home/coq/${CONTRIB_NAME}
```

Comment:
Hi @proux01 sorry I didn't realize that beforehand, but this second occurrence of `sudo chown` is unnecessary (it has already be done so that the UID of the files brought by GitHub are adapted w.r.t. the UID of the coq user in the image)
But this is a detail, it will work with or without this line anyway :)

Reply:
Indeed, thanks!